README.md

```diff
@@ -17,41 +17,53 @@
 </p>
 
 
-Transformers Interpret is a model explainability tool designed to work exclusively with 🤗 transformers.
+Transformers Interpret is a model explainability tool designed to work exclusively with the [transformers][transformers] package.
 
-In line with the philosophy of the transformers package tranformers interpret allows any transformers model to be explained in just two lines. It even supports visualizations in both notebooks and as savable html files.
+In line with the philosophy of the transformers package Tranformers Interpret allows any transformers model to be explained in just two lines. It even supports visualizations in both notebooks and as savable html files.
 
-This package stands on the shoulders of the the incredible work being done by the teams at [Pytorch Captum](https://captum.ai/) and [Hugging Face](https://huggingface.co/) and would not exist if not for the amazing job they are both doing in the fields of nlp and model interpretability respectively.
 
 ## Install
 
-I recommend Python 3.6 or higher, Pytorch 1.5.0 or higher, transformers v3.0.0 or higher, and captum 0.3.1 (**required**). The package does not work with Python 2.7.
-
-```bash
+```posh
 pip install transformers-interpret
 ```
 
+Required:
+
+* Python 3.6 or higher
+* Pytorch 1.5.0 or higher
+* [transformers][transformers] v3.0.0 or higher
+* captum 0.3.1 
+
+The package does not work with Python 2.7 or below.
+
+
+
+# Documentation
```

Comment:
I'd state that this is the documentation, to avoid users looking for one that doesn't exist.